src/Mod/Sketcher/Gui/EditModeConstraintCoinManager.cpp

```diff
@@ -818,51 +833,61 @@ void EditModeConstraintCoinManager::processConstraints(const GeoListFacade& geol
                     else if (Constr->Second != GeoEnum::GeoUndef) {
                         const Part::Geometry* geo =
                             geolistfacade.getGeometryFromGeoId(Constr->Second);
+                        const Part::Geometry* geo1 =
+                            geolistfacade.getGeometryFromGeoId(Constr->First);
                         if (geo->getTypeId() == Part::GeomLineSegment::getClassTypeId()) {
-                            const Part::GeomLineSegment* lineSeg =
-                                static_cast<const Part::GeomLineSegment*>(geo);
-                            Base::Vector3d l2p1 = lineSeg->getStartPoint();
-                            Base::Vector3d l2p2 = lineSeg->getEndPoint();
-                            if (Constr->FirstPos
+                            if (Constr->SecondPos

```

Comment:
I don't like the line breaks like these, maybe it's personal but I feel they make the code harder to read.

Reply:
It's configured in https://github.com/FreeCAD/FreeCAD/blob/master/.clang-format